src/components/common/Members/Members.css

```diff
@@ -0,0 +1,109 @@
+@value query700 from '~styles/queries.css';
+
+.membersContainer {
+  display: flex;
+}
+
+.main {
+  margin-top: 20px;
+  min-width: 618px;
+}
+
+.main table .tableBody {
+  overflow-y: auto;
+  overflow-x: none;
+}
+
+.main table .tableBody tr {
+  overflow-x: auto;
+}
+
+.communityRole {
+  font-size: var(--size-tiny);
+  font-weight: var(--weight-bold);
+  white-space: nowrap;
+}
+
+.subscribeCallToAction {
```

Comment:
Some of this css doesn't appear to be used in `Members.tsx`, e.g. subscribeCallToAction, subscribeButton, subscribedIcon ...